packages/js/src/generators/library/library.ts

```diff
@@ -72,6 +73,12 @@ export async function projectGenerator(
 
   tasks.push(addProjectDependencies(tree, options));
 
+  if (options.rootProject && options.publishable) {
+    tasks.push(
+      addDependenciesToPackageJson(tree, {}, { verdaccio: verdaccioVersion })
+    );
+  }
```

Comment:
I don't think this quite makes sense. We should add the verdaccio setup even if the lib we are generating isn't the root project, I'm just saying that I think the task itself should be added to the root project rather than the plugin project s.t. multiple plugins don't end up starting multiple registries.

Reply:
changed